certified-connectors/EBMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,481 @@
+                  "type": "string",
+                  "description": "A multi-line memo",
+                  "title": "Memo"
+                },
+                "UPC": {
+                  "type": "string",
+                  "description": "UPC",
+                  "title": "UPC"
+                },
+                "MFG": {
+                  "type": "string",
+                  "description": "Must match EBMS Manufacturer's ID exactly",
+                  "title": "Mfg ID",
+                  "x-ms-visibility": "advanced"
+                },
+                "MFG_PART": {
+                  "type": "string",
+                  "description": "Manufacturer's Part Number",
+                  "title": "MFG Part",
+                  "x-ms-visibility": "advanced"
+                },
+                "PRI_VENDOR": {
+                  "type": "string",
+                  "description": "The Primary Vendor ID must match EBMS's ID exactly",
+                  "title": "Primary Vendor ID",
+                  "x-ms-visibility": "advanced"
+                },
+                "EACH_UNIT": {
+                  "type": "string",
+                  "description": "Main unit of measure",
+                  "title": "Main Unit"
+                },
+                "WEIGHT": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "Weight",
+                  "title": "Weight",
+                  "x-ms-visibility": "advanced"
+                },
+                "COST": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "Cost",
+                  "title": "Cost"
+                },
+                "BASE": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "Base Price",
+                  "title": "Base Price"
+                },
+                "EXTERNALID": {
+                  "type": "string",
+                  "description": "Unique ID of the program integrated with EBMS",
+                  "title": "External ID",
+                  "x-ms-visibility": "advanced"
+                }
+              }
+            }
+          }
+        ],
+        "description": "This action will update an exising EBMS product."
+      }
+    },
+    "/INVENTRE": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "List of inventory folders.",
+            "schema": {
+              "$ref": "#/definitions/ListProductCategoriesResponse"
+            }
+          }
+        },
+        "summary": "Get Product Category ID",
+        "description": "This will get all the Product Categories.",
+        "operationId": "GetProductCategoryID",
+        "parameters": [],
+        "x-ms-visibility": "internal"
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://eaglebusinesssoftware.com"
```

Comment:
@bradleymarshall1982 the website URL should be a valid URL. please change to "https://www.eaglebusinesssoftware.com"

Reply:
Fixed.